sycl/source/detail/error_handling/enqueue_kernel.cpp

```diff
@@ -268,6 +268,31 @@ bool handleInvalidWorkItemSize(const device_impl &DeviceImpl,
   return 0;
 }
 
+bool handleInvalidValue(const device_impl &DeviceImpl,
+                        const NDRDescT &NDRDesc) {
+  const plugin &Plugin = DeviceImpl.getPlugin();
+  RT::PiDevice Device = DeviceImpl.getHandleRef();
+
+  size_t MaxNWGs[] = {0, 0, 0};
+  Plugin.call<PiApiKind::piDeviceGetInfo>(
+      Device, PI_EXT_ONEAPI_DEVICE_INFO_MAX_WORK_GROUPS_3D, sizeof(MaxNWGs),
+      &MaxNWGs, nullptr);
+  for (unsigned int I = 0; I < NDRDesc.Dims; I++) {
+    size_t n_wgs = NDRDesc.GlobalSize[I] / NDRDesc.LocalSize[I];
```

Comment:
Nit: The name of this variable is not in line with the naming of the surrounding code.